electron/src/plugins.ts

```diff
@@ -10,7 +10,7 @@ export * from '@capacitor/core/dist/esm/web/storage';
 export * from './electron/filesystem';
 export * from './electron/network';
 export * from './electron/splashscreen';
-export * from './electron/toast';
+export * from '@capacitor/core/dist/esm/web/toast';
```

Comment:
do we even need to export this then? The idea is that web implementations should just work if they are the sole implementation of a given plugin at runtime

Reply:
I don't know for sure, but the other web plugins are exported here, so I did the same for toast, I can try to remove all of them and check if they continue working

Reply:
Yeah, it's not needed, I've removed all of them.